src/ui/rendering/handlebarsrenderer.js

```diff
@@ -207,5 +207,9 @@ export default class HandlebarsRenderer extends Renderer {
       return getInverted ? self.SafeString(highlightedVal.getInverted())
         : self.SafeString(highlightedVal.get());
     });
+
+    self.registerHelper('renderTemplate', function (template) {
```

Comment:
Do we need this helper? It's a bit weird to have a HBS helper that compiles and renders an arbitrary HBS template. That seems contrary to the conventional flow of things. Is there a way to avoid this? Could the results header component have something like:

this._compiledCountTemplate = this._renderer.compile(this._resultsCountTemplate);

Then in the HBS we'd have:

{{#if compiledCountTemplate}}
  {{ > compiledCountTemplate }}
{{/if}}

Reply:
Sure it probably doesn't make sense to recompile the template every time